.changeset/stupid-plants-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": patch
+---
+
+fix(web): Correct formatWei precision for large decimal values
```

Comment:
```suggestion
Corrected formatWei precision for large decimal values
```
_[picky]_

I've checked other changeset entries:
- the prefix is not added
- we use the past verb